<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
<title>EMBER - Fig 3.5</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="../screen.css" rel="stylesheet" type="text/css">
</head>

<body>
<br>
<table width="75%" border="0" align="center" cellpadding="0" cellspacing="0">
  <tr>
    <td>
      <div align="left" class="photocaption"><img src="files/emotifgroups.gif" width="600" height="236"><br>
          <br>
        Fig 3.5. eMOTIF substitution groups. These are found to occur together 
        at a significant frequency in columns of aligned sequences, for example 
        as observed in the Blocks databases. They have been arranged hierarchically 
        and coloured with the standard amino acid 7-colour scheme to illustrate 
        the relationship between conservation and amino acid properties. Note 
        the overlap between aromatic and aliphatic groups, as well as between 
        basic and neutral groups. Adapted from Nevill-Manning, C.G., Wu, T.D., 
        Brutlag, D.L. Highly specific protein sequence motifs for genome analysis. 
        1998 <em>Proc. Natl. Acad. Sci. USA. </em>95:11. pp:5865-5871 - Copyright 
        (1998) National Academy of Sciences, USA.</div></td>
  </tr>
</table>
<br><p><div align="center"><form><input class="formbutton" type="button" value="Close Window" +  onclick="self.close()"></form></div><p></p>

<p>&nbsp;</p></body>
</html>

