<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<!-- saved from url=(0049)http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00017 -->
<HTML><HEAD><TITLE>NiceSite View of PROSITE: PDOC00017 (documentation file)</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1252">
<META content="MSHTML 6.00.2713.1100" name=GENERATOR></HEAD>
<BODY text=#000000 bgColor=#ffffff>
<div align="center">
  <table border="1">
    <tr> 
      <td> <hr size=2> <center>
          <table cellspacing=0 cellpadding=2 width="100%" align=center border=2>
            <tbody>
              <tr> 
                <td align=middle bgcolor=#ffffff><b><a href="http://us.expasy.org/" 
      target=_top>ExPASy Home page</a></b></td>
                <td align=middle bgcolor=#ff9999><b><a 
      href="http://us.expasy.org/sitemap.html" target=_top>Site Map</a></b></td>
                <td align=middle bgcolor=#c0eac0><b><a 
      href="http://us.expasy.org/ExpasyHunt/" target=_top>Search ExPASy</a></b></td>
                <td align=middle bgcolor=#ccccff><b><a 
      href="http://us.expasy.org/contact.html" target=_top>Contact us</a></b></td>
                <td align=middle bgcolor=#33ccff><b><a 
      href="http://us.expasy.org/prosite/" 
  target=_top>PROSITE</a></b></td>
              </tr>
            </tbody>
          </table>
        </center>
        <center>
          <table border=1>
            <tbody>
              <tr> 
                <td bgcolor=#ffffcc><a href="http://www.ncsc.org/">Hosted by NCSC 
                  US</a></td>
                <td>Mirror sites:</td>
                <td><a 
    href="http://ca.expasy.org/cgi-bin/nicedoc.pl?PDOC00017">Canada</a></td>
                <td><a 
    href="http://cn.expasy.org/cgi-bin/nicedoc.pl?PDOC00017">China</a></td>
                <td><a 
    href="http://kr.expasy.org/cgi-bin/nicedoc.pl?PDOC00017">Korea</a></td>
                <td><a 
      href="http://www.expasy.org/cgi-bin/nicedoc.pl?PDOC00017">Switzerland</a></td>
                <td><a 
    href="http://tw.expasy.org/cgi-bin/nicedoc.pl?PDOC00017">Taiwan</a></td>
              </tr>
            </tbody>
          </table>
        </center>
        <hr size=2> <h1>NiceSite View of PROSITE: <a 
href="http://us.expasy.org/cgi-bin/get-prodoc-entry?PDOC00017">PDOC00017</a> (documentation)</h1>
        <h1>ATP/GTP-binding site motif A (P-loop)</h1>
        <table bgcolor=#afdfff border=1>
          <tbody>
            <tr> 
              <td bgcolor=#0000dd colspan=2><font color=#ffffff><b>PROSITE cross-reference(s)</b></font></td>
            </tr>
            <tr> 
              <td colspan=2><a 
      href="http://us.expasy.org/cgi-bin/nicesite.pl?PS00017">PS00017; ATP_GTP_A 
                </a><br></td>
            </tr>
            <tr> 
              <td bgcolor=#0000dd colspan=2><font 
      color=#ffffff><b>Documentation</b></font></td>
            </tr>
            <tr> 
              <td colspan=2> <pre>From sequence comparisons and crystallographic data analysis it has been shown
[1,2,3,4,5,6] that an appreciable proportion of proteins that  bind ATP or GTP
share a number of more or less conserved sequence motifs.   The best conserved
of these  motifs  is  a  glycine-rich region, which typically forms a flexible
loop between a &beta;-strand and an &alpha;-helix. This loop interacts with one of
the phosphate  groups  of  the  nucleotide.   This sequence motif is generally
referred to as the 'A' consensus sequence [1] or the 'P-loop' [5].

There are numerous ATP- or GTP-binding proteins  in which the P-loop is found.
We list below  a number of protein  families  for  which  the relevance of the
presence of such motif has been noted:

 - ATP synthase &alpha; and &beta; subunits (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00137">PDOC00137</a>&gt;).
 - Myosin heavy chains.
 - Kinesin heavy chains and kinesin-like proteins (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00343">PDOC00343</a>&gt;).
 - Dynamins and dynamin-like proteins (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00362">PDOC00362</a>&gt;).
 - Guanylate kinase (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00670">PDOC00670</a>&gt;).
 - Thymidine kinase (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00524">PDOC00524</a>&gt;).
 - Thymidylate kinase (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC01034">PDOC01034</a>&gt;).
 - Shikimate kinase (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00868">PDOC00868</a>&gt;).
 - Nitrogenase iron protein family (nifH/chlL) (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00580">PDOC00580</a>&gt;).
 - ATP-binding proteins involved  in 'active transport' (ABC transporters) [7]
   (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00185">PDOC00185</a>&gt;).
 - DNA and RNA helicases [8,9,10].
 - GTP-binding elongation factors (EF-Tu, EF-1 &alpha;, EF-G, EF-2, etc.).
 - Ras family of GTP-binding proteins (Ras, Rho, Rab, Ral, Ypt1, SEC4, etc.).
 - Nuclear protein ran (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00859">PDOC00859</a>&gt;).
 - ADP-ribosylation factors family (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00781">PDOC00781</a>&gt;).
 - Bacterial dnaA protein (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00771">PDOC00771</a>&gt;).
 - Bacterial recA protein (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00131">PDOC00131</a>&gt;).
 - Bacterial recF protein (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00539">PDOC00539</a>&gt;).
 - Guanine nucleotide-binding proteins &alpha; subunits (Gi, Gs, Gt, G0, etc.).
 - DNA mismatch repair proteins mutS family (See &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00388">PDOC00388</a>&gt;).
 - Bacterial type II secretion system protein E (see &lt;<a href="http://us.expasy.org/cgi-bin/nicedoc.pl?PDOC00567">PDOC00567</a>&gt;).

Not all ATP- or GTP-binding proteins are picked-up by this motif.  A number of
proteins escape detection because the structure   of their ATP-binding site is
completely different from that of the P-loop.  Examples  of  such proteins are
the E1-E2 ATPases or  the  glycolytic kinases.   In  other ATP- or GTP-binding
proteins the flexible loop exists  in a  slightly different form; this is  the
case for tubulins or protein kinases.  A special mention must  be reserved for
adenylate  kinase,  in  which  there  is a  single  deviation  from the P-loop
pattern: in the last position Gly is found instead of Ser or Thr.

</pre></td>
            </tr>
            <tr> 
              <td bgcolor=#0000dd colspan=2><font color=#ffffff><b>Description 
                of pattern(s) and/or profile(s)</b></font></td>
            </tr>
            <tr> 
              <td bgcolor=#33ccff colspan=2></td>
            </tr>
            <tr bgcolor=#33ccff> 
              <td><b>Consensus pattern</b></td>
              <td>[AG]-x(4)-G-K-[ST]</td>
            </tr>
            <tr> 
              <td><b>Sequences known to belong to this class detected by the pattern</b></td>
              <td>a majority.</td>
            </tr>
            <tr> 
              <td><b>Other sequence(s) detected in SWISS-PROT</b></td>
              <td>in addition to the proteins listed above, the 'A' motif is also 
                found in a number of other proteins. Most of these proteins probably 
                bind a nucleotide, but others are definitively not ATP- or GTP-binding 
                (as for example chymotrypsin, or human ferritin light chain).</td>
            </tr>
            <tr> 
              <td bgcolor=#0000dd colspan=2><font color=#ffffff><b>Expert(s) to 
                contact by email<b></b></b></font></td>
            </tr>
            <tr> 
              <td>Koonin E.V.</td>
              <td><a 
      href="http://us.expasy.org/cgi-bin/mailform/exp?Koonin_E.V.&amp;_ATP/GTP-binding_site_motif_A_(P-loop)&amp;koonin@ncbi.nlm.nih.gov">http://us.expasy.org/cgi-bin/mailform/exp?Koonin_E.V.&amp;_ATP/GTP-binding_site_motif_A_(P-loop)&amp;koonin@ncbi.nlm.nih.gov</a><br></td>
            </tr>
            <tr> 
              <td bgcolor=#0000dd colspan=2><font color=#ffffff><b>Last update<b></b></b></font></td>
            </tr>
            <tr> 
              <td colspan=2>July 1999 / Text revised.</td>
            </tr>
            <tr> 
              <td bgcolor=#0000dd colspan=2><font 
    color=#ffffff><b>References</b></font></td>
            </tr>
            <tr> 
              <td colspan=2>[ 1]<br>
                Walker J.E., Saraste M., Runswick M.J., Gay N.J. <br> <a 
      href="http://emboj.oupjournals.org/cgi/lookup?vol=1&amp;fp=945&amp;view=abstract">EMBO 
                J. 1:945-951(1982). </a><br></td>
            </tr>
            <tr> 
              <td colspan=2>[ 2]<br>
                Moller W., Amons R. <br>
                FEBS Lett. 186:1-7(1985). <br></td>
            </tr>
            <tr> 
              <td colspan=2>[ 3]<br>
                Fry D.C., Kuby S.A., Mildvan A.S. <br> <a 
      href="http://www.pnas.org/cgi/lookup?vol=83&amp;fp=907&amp;view=abstract">Proc. 
                Natl. Acad. Sci. U.S.A. 83:907-911(1986). </a><br></td>
            </tr>
            <tr> 
              <td colspan=2>[ 4]<br>
                Dever T.E., Glynias M.J., Merrick W.C. <br> <a 
      href="http://www.pnas.org/cgi/lookup?vol=84&amp;fp=1814&amp;view=abstract">Proc. 
                Natl. Acad. Sci. U.S.A. 84:1814-1818(1987). </a><br></td>
            </tr>
            <tr> 
              <td colspan=2>[ 5]<br>
                Saraste M., Sibbald P.R., Wittinghofer A. <br>
                Trends Biochem. Sci. 15:430-434(1990). <br></td>
            </tr>
            <tr> 
              <td colspan=2>[ 6]<br>
                Koonin E.V. <br>
                J. Mol. Biol. 229:1165-1174(1993). <br></td>
            </tr>
            <tr> 
              <td colspan=2>[ 7]<br>
                Higgins C.F., Hyde S.C., Mimmack M.M., Gileadi U., Gill D.R., 
                Gallagher M.P. <br>
                J. Bioenerg. Biomembr. 22:571-592(1990). <br></td>
            </tr>
            <tr> 
              <td colspan=2>[ 8]<br>
                Hodgman T.C. <br>
                Nature 333:22-23(1988) and Nature 333:578-578(1988) (Errata). 
                <br></td>
            </tr>
            <tr> 
              <td colspan=2>[ 9]<br>
                Linder P., Lasko P., Ashburner M., Leroy P., Nielsen P.J., Nishi 
                K., Schnier J., Slonimski P.P. <br>
                Nature 337:121-122(1989). <br></td>
            </tr>
            <tr> 
              <td colspan=2>[10]<br>
                Gorbalenya A.E., Koonin E.V., Donchenko A.P., Blinov V.M. <br> 
                <a 
      href="http://nar.oupjournals.org/cgi/lookup?vol=17&amp;fp=4713&amp;view=abstract">Nucleic 
                Acids Res. 17:4713-4730(1989). </a><br></td>
            </tr>
            <tr> 
              <td bgcolor=#0000dd colspan=2><font 
    color=#ffffff><b>Copyright</b></font></td>
            </tr>
            <tr> 
              <td colspan=2><font size=-1>This PROSITE entry is copyright by the 
                Swiss Institute of Bioinformatics (SIB). There are no restrictions 
                on its use by non-profit institutions as long as its content is 
                in no way modified and this statement is not removed. Usage by 
                and for commercial entities requires a license agreement (See 
                <a 
      href="http://www.isb-sib.ch/announce/">http://www.isb-sib.ch/announce/</a> 
                or email to <a href="mailto:license@isb-sib.ch">license@isb-sib.ch</a>). 
                </font></td>
            </tr>
          </tbody>
        </table>
        <p> 
        <address>
        <a href="http://us.expasy.org/cgi-bin/get-prodoc-entry?PDOC00017">View 
        entry in original PROSITE document format</a>
        </address>
        <address>
        <a 
href="http://us.expasy.org/cgi-bin/get-prosite-raw.pl?PDOC00017">View entry in 
        raw text format (no links)</a>
        </address>
        <hr size=2> <center>
          <table cellspacing=0 cellpadding=2 width="100%" align=center border=2>
            <tbody>
              <tr> 
                <td align=middle bgcolor=#ffffff><b><a href="http://us.expasy.org/" 
      target=_top>ExPASy Home page</a></b></td>
                <td align=middle bgcolor=#ff9999><b><a 
      href="http://us.expasy.org/sitemap.html" target=_top>Site Map</a></b></td>
                <td align=middle bgcolor=#c0eac0><b><a 
      href="http://us.expasy.org/ExpasyHunt/" target=_top>Search ExPASy</a></b></td>
                <td align=middle bgcolor=#ccccff><b><a 
      href="http://us.expasy.org/contact.html" target=_top>Contact us</a></b></td>
                <td align=middle bgcolor=#33ccff><b><a 
      href="http://us.expasy.org/prosite/" 
  target=_top>PROSITE</a></b></td>
              </tr>
            </tbody>
          </table>
        </center>
        <center>
          <table border=1>
            <tbody>
              <tr> 
                <td bgcolor=#ffffcc><a href="http://www.ncsc.org/">Hosted by NCSC 
                  US</a></td>
                <td>Mirror sites:</td>
                <td><a 
    href="http://ca.expasy.org/cgi-bin/nicedoc.pl?PDOC00017">Canada</a></td>
                <td><a 
    href="http://cn.expasy.org/cgi-bin/nicedoc.pl?PDOC00017">China</a></td>
                <td><a 
    href="http://kr.expasy.org/cgi-bin/nicedoc.pl?PDOC00017">Korea</a></td>
                <td><a 
      href="http://www.expasy.org/cgi-bin/nicedoc.pl?PDOC00017">Switzerland</a></td>
                <td><a 
    href="http://tw.expasy.org/cgi-bin/nicedoc.pl?PDOC00017">Taiwan</a></td>
              </tr>
            </tbody>
          </table>
        </center>
        <hr size=2></td>
    </tr>
  </table>
</div>
<p align="center"><font face="Arial, Helvetica, sans-serif" size="2"><b>Typical 
  PROSITE documentation entry.</b></font></p>
<form>
  <div align="center">
    <input type="button" value="CLOSE" onClick="window.close()" name="button">
  </div>
</form>
<p align="center"></p>
<p>&nbsp;</p></body></HTML>
