<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"><html>
<head>
<title>EMBER: Chapter 11 - Investigating the inositol phosphatase family: References</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<!-- Fireworks MX Dreamweaver MX target. Created Wed Oct 16 13:14:05 GMT-2200 2002-->
<script language="JavaScript" type="text/JavaScript">
<!-- 

function MM_preloadImages() { //v3.0
 var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
  var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
  if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

//-->
</script>
<script language="JavaScript" type="text/JavaScript" src="scripts.js">
</script>
<link href="screen.css" rel="stylesheet" type="text/css">
</head>
<body bgcolor="#9dafd1" background="Graphics/background_left.jpg" onLoad="MM_preloadImages('Graphics/banner_top_r2_c2_f2.jpg')">
<?PHP
	include "notepad.php";
	?>
<div style="z-index:1; position:absolute; left:0px; top:0px">
<table width="750" border="0" cellpadding="0" cellspacing="0"><!--DWLayoutTable-->
 <tr>
  <td colspan="2">
	<?PHP
	include "banner.php";
	?></td>
 </tr>
 <tr>
 <td width="215" height="530" valign="top" nowrap style=" background-image:  url(Graphics/backgrounds/ch11.jpg); background-repeat: no-repeat"></td>
  <td align="left" valign="top">
	<table width="100%" border="0" cellpadding="0" cellspacing="0">
  <tr> 
   <td valign="top"> 
    <table border=0 cellspacing=0 cellpadding=0 width="100%" height=21> <tr> 
      <td valign=top height=21><img src="Graphics/tab_refs_air.gif" width="314" height=21 border=0 align=bottom usemap="#Map"></td>
      <td valign=top height=21 width="100%"> 
          <table border=0 cellspacing=0 cellpadding=0 width="100%" height="100%">
        <tr> 
         <td valign=top height=20 width="100%"> 
          <p>&nbsp;</p></td>
        </tr>
        <tr> 
         <td valign=top height=1 bgcolor="#f44d1e"><img src="Graphics/spacer.gif" width=1 height=1 border=0 align=bottom></td>
        </tr>
       </table></td>
     </tr>
    </table>
    <table border=0 cellspacing=0 cellpadding=0>
     <tr> 
      <td valign=top width=1 bgcolor="#f44d1e"> 
       <p><img src="Graphics/spacer.gif" width=1 height=1 align=bottom></p></td>
      <td valign=top background="Graphics/background_content.gif"> 
       
			 
			 
			 
			 <table border=0 cellspacing=0 cellpadding=10 width="100%">
        <tr> 
          <td valign=top> 
           <?PHP
include "refs_header.php";
?>
<p>Aiyar, A. <BR>
<b>The use of CLUSTAL W and CLUSTAL X for multiple sequence alignment</b>. <BR>
2000.<i>Methods Mol Biol</i>. 132:pp: 221-41. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10547838 target=new>pubmed</a> <BR>
<BR>

	Attwood, T. K. <BR>
<b>The PRINTS database: a resource for identification of protein families</b>. <BR>
2002.<i>Brief Bioinform</i>. 3:3. pp: 252-63. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12230034 target=new>pubmed</a> <BR>
<BR>

	Attwood, T. K., Blythe, M. J., Flower, D. R., Gaulton, A., Mabey, J. E., Maudling, N., McGregor, L., Mitchell, A. L., Moulton, G., Paine, K. and Scordis, P. <BR>
<b>PRINTS and PRINTS-S shed light on protein ancestry</b>. <BR>
2002.<i>Nucleic Acids Res</i>. 30:1. pp: 239-41. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11752304 target=new>pubmed</a> <BR>
<BR>

	Bateman, A., Birney, E., Durbin, R., Eddy, S. R., Finn, R. D. and Sonnhammer, E. L. <BR>
<b>Pfam 3.1: 1313 multiple alignments and profile HMMs match the majority of proteins.</b> <BR>
1999.<i>Nucleic Acids Res</i>. 27:1. pp: 260-2. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9847196 target=new>pubmed</a> <BR>
<BR>

	Bateman, A., Birney, E., Durbin, R., Eddy, S. R., Howe, K. L. and Sonnhammer, E. L. <BR>
<b>The Pfam protein families database</b>. <BR>
2000.<i>Nucleic Acids Res</i>. 28:1. pp: 263-6. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10592242 target=new>pubmed</a> <BR>
<BR>

	Bateman, A., Birney, E., Cerruti, L., Durbin, R., Etwiller, L., Eddy, S. R., Griffiths-Jones, S., Howe, K. L., Marshall, M. and Sonnhammer, E. L. <BR>
<b>The Pfam protein families database</b>. <BR>
2002.<i>Nucleic Acids Res</i>. 30:1. pp: 276-80. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11752314 target=new>pubmed</a> <BR>
<BR>

	Falquet, L., Pagni, M., Bucher, P., Hulo, N., Sigrist, C. J., Hofmann, K. and Bairoch, A. <BR>
<b>The PROSITE database, its status in 2002</b>. <BR>
2002.<i>Nucleic Acids Res</i>. 30:1. pp: 235-8. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11752303 target=new>pubmed</a> <BR>
<BR>

	Needleman, S. B. and Wunsch, C. D. <BR>
<b>A general method applicable to the search for similarities in the amino acid sequence of two proteins.</b> <BR>
1970.<i>J Mol Biol</i>. 48:3. pp: 443-53. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=5420325 target=new>pubmed</a> <BR>
<BR>

	Sigrist, C. J., Cerutti, L., Hulo, N., Gattiker, A., Falquet, L., Pagni, M., Bairoch, A. and Bucher, P. <BR>
<b>PROSITE: a documented database using patterns and profiles as motif descriptors.</b> <BR>
2002.<i>Brief Bioinform</i>. 3:3. pp: 265-74. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12230035 target=new>pubmed</a> <BR>
<BR>

	Sonnhammer, E. L., Eddy, S. R. and Durbin, R. <BR>
<b>Pfam: a comprehensive database of protein domain families based on seed alignments.</b> <BR>
1997.<i>Proteins</i>. 28:3. pp: 405-20. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9223186 target=new>pubmed</a> <BR>
<BR>

	Sonnhammer, E. L., Eddy, S. R., Birney, E., Bateman, A. and Durbin, R. <BR>
<b>Pfam: multiple sequence alignments and HMM-profiles of protein domains</b>. <BR>
1998.<i>Nucleic Acids Res</i>. 26:1. pp: 320-2. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9399864 target=new>pubmed</a> <BR>
<BR>

	Thompson, J. D., Higgins, D. G. and Gibson, T. J. <BR>
<b>CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.</b> <BR>
1994.<i>Nucleic Acids Res</i>. 22:22. pp: 4673-80. <BR>
 <a href=http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7984417 target=new>pubmed</a></p></td>
        </tr>
       </table></td>
      <td valign=top width=1 bgcolor="#f44d1e"> 
       <p><img src="Graphics/spacer.gif" width=1 height=1 align=bottom></p></td>
     </tr>
     <tr> 
      <td valign=top width="1" bgcolor="#f44d1e"> 
       <p><img src="Graphics/spacer.gif" width=1 height=1 align=bottom></p></td>
      <td valign=top width="100%" bgcolor="#f44d1e"> 
       <p><img src="Graphics/spacer.gif" width=1 height=1 align=bottom></p></td>
      <td valign=top width="1" bgcolor="#f44d1e"> 
       <p><img src="Graphics/spacer.gif" width=1 height=1 align=bottom></p></td>
     </tr>
    </table></td>
   <td width=15 valign="top">&nbsp;</td>
  </tr>
 </table></td>
 </tr>
</table>

 <map name="Map">
  <area shape="rect" coords="6,4,39,18" href="ch11_aims.php" alt="Aims" title="Aims">
  <area shape="rect" coords="56,4,122,18" href="ch11_1_instr.php" alt="Instructions" title="Instructions">
  <area shape="rect" coords="138,5,200,18" href="ch11_refs.php" alt="References" title="References">
 </map> 
 

</div>
<script>awmSelectedItem=18</script>
<?PHP
	include "toc.php";
	?><p>&nbsp;</p></body>
</html>
